### ISSUE TITLE
Double tab  autocompletion  shortcut

### ISSUE THREAD

**Issue by ggrrll** [jupyter/notebook#2299]
HI, 

this is an idea for enhancement:

why not using the fact that you can press more times <kbd> tab </kbd> , after the autocompletion window appears? 

For instance, in Rstudio, by pressing a 2nd time you insert the 1st suggestion of the list (this is very handy). 
Another possibility would be to use <kbd> tab </kbd> to cycle through the suggestions.

**Comment by TheSarfaraz** [comment]
Yes second time pressing tab should just select and insert the selected suggestion

**Comment by ggrrll** [comment]
@TheSarfaraz  sorry, are you talking about a specific package? -- this is not happening for me at the moment...

- on notebook 5.7.4

**Comment by TheSarfaraz** [comment]
> @TheSarfaraz sorry, are you talking about a specific package? -- this is not happening for me at the moment...
> 
> actually, for me (for instance on matplotlib) the 2nd tab even deletes what the first tab shows
> 
> ps: on notebook 5.7.4

It's not happening, I'm suggesting this feature that it should work this way ...

In any good IDEs we can press tab to see the list and again press tab to insert the suggestion, same should be implemented in the Jupyter as well.
